### PR TITLE
Clarify that the API works in Python 3.

### PR DESCRIPTION
**Pull request by jkinkead**
Also add a passive-aggressive link to the closed issue.

Slightly tongue-in-cheek, but issue #700 really should be reopened.

### CODE REVIEW

File: tensorflow_serving/g3doc/setup.md [hunk 1]
```diff
@@ -72,7 +72,12 @@ the `tensorflow-serving-api` PIP package using:
 pip install tensorflow-serving-api
 ```
 
-Note: The TensorFlow Serving Python API only supports Python 2.7. It does not support Python 3.
+Note: The TensorFlow Serving Python API
+[is only published for Python 2](https://pypi.python.org/pypi/tensorflow-serving-api/1.0.0),
```
Comment (kirilg) [inline tensorflow_serving/g3doc/setup.md]: Let's link to the package, not a specific version. Just https://pypi.python.org/pypi/tensorflow-serving-api

File: tensorflow_serving/g3doc/setup.md [hunk 2]
```diff
@@ -72,7 +72,12 @@ the `tensorflow-serving-api` PIP package using:
...
+but will work for Python 3 if you either build it yourself, or download the Python 2 version,
+unzip it, and copy it into your Python 3 path. There was
```
Comment (kirilg) [inline tensorflow_serving/g3doc/setup.md]: I reopened the issue. Let's remove the last line "There was [an issue tracking publishing the API as Python 3](https://github.com/tensorflow/serving/issues/700), but it was closed."

Comment (jkinkead) [reply tensorflow_serving/g3doc/setup.md]: Would you like a link to the issue, without the snark, or should I just leave it off?

Comment (kirilg) [reply tensorflow_serving/g3doc/setup.md]: I'm ok with linking to the issue as a feature request if you'd like. e.g. add something like:
"There's a [feature request](https://github.com/tensorflow/serving/issues/700) to publish a Python 3 package as well".

**Comment by jkinkead** [comment]
Thanks, comments addressed!

**Comment by kirilg** [comment]
Thanks! Merging now

**Pull request merged**